cmd/juju/cloud/show.go

```diff
@@ -87,9 +87,10 @@ func (c *showCloudCommand) cloudAPI() (showCloudAPI, error) {
 
 func (c *showCloudCommand) SetFlags(f *gnuflag.FlagSet) {
 	c.OptionalControllerCommand.SetFlags(f)
-	// We only support yaml for display purposes.
-	c.out.AddFlags(f, "yaml", map[string]cmd.Formatter{
-		"yaml": cmd.FormatYaml,
+	c.out.AddFlags(f, "display", map[string]cmd.Formatter{
+		"yaml":    cmd.FormatYaml,
+		"json":    cmd.FormatJson,
+		"display": cmd.FormatYaml,
```

Comment:
`display` is `tabular` other places.

Reply:
I'd agree if we had a tabular output. I think with @tmihoc changes she is thinking about could actually be tabular. Maybe we just call it tabular and update the output to be more tabular.